config/routes.rb

```diff
@@ -314,6 +314,7 @@
   get "/new/:template" => "articles#new"
 
   get "/pod" => "podcast_episodes#index"
+  get "/podcasts" => "podcast_episodes#index"
```

Comment:
Right now https://dev.to/podcasts is a 404 page, I thought it might be nice to make it work

Reply:
Is it right to have 2 different urls for the same page? Maybe there should be a redirect.

Reply:
Great point! I think you're right, maybe @benhalpern can chime in on this

Reply:
Yeah, we should make this a redirect.

Reply:
Is it better if "/podcasts" redirects to "/pod" or the other way around?

Reply:
"/podcasts" should redirect to "/pod", which is what we've been treating as the canonical URL so far.